### PR TITLE
Fix readme

### PR DESCRIPTION
**Pull request by bwagner5**
*Issue #, if available:*
N/A

*Description of changes:*
- Sync Readme changes
- Remove log from test code

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,17 +145,17 @@ Filter Flags:
       --deny-list string                  List of instance types which should be excluded w/ regex syntax (Example: m[1-2]\.*)
   -e, --ena-support                       Instance types where ENA is supported or required
   -f, --fpga-support                      FPGA instance types
-      --gpu-memory-total float            Number of GPUs' total memory in GiB (Example: 4) (sets --gpu-memory-total-min and -max to the same value)
-      --gpu-memory-total-max float        Maximum Number of GPUs' total memory in GiB (Example: 4) If --gpu-memory-total-min is not specified, the lower bound will be 0
-      --gpu-memory-total-min float        Minimum Number of GPUs' total memory in GiB (Example: 4) If --gpu-memory-total-max is not specified, the upper bound will be infinity
+      --gpu-memory-total string           Number of GPUs' total memory (Example: 4 GiB) (sets --gpu-memory-total-min and -max to the same value)
```
Comment (brycahta) [inline README.md:148]: Will the CLI parse the space between `4` and `GiB` ?

File: README.md [hunk 2]
```diff
@@ -145,17 +145,17 @@ Filter Flags:
...
-      --memory-min float                  Minimum Amount of Memory available in GiB (Example: 4) If --memory-max is not specified, the upper bound will be infinity
+  -m, --memory string                     Amount of Memory available (Example: 4 GiB) (sets --memory-min and -max to the same value)
+      --memory-max string                 Maximum Amount of Memory available (Example: 4 GiB) If --memory-min is not specified, the lower bound will be 0
+      --memory-min string                 Minimum Amount of Memory available (Example: 4 GiB) If --memory-max is not specified, the upper bound will be infinity
       --network-interfaces int            Number of network interfaces (ENIs) that can be attached to the instance (sets --network-interfaces-min and -max to the same value)
       --network-interfaces-max int        Maximum Number of network interfaces (ENIs) that can be attached to the instance If --network-interfaces-min is not specified, the lower bound will be 0
       --network-interfaces-min int        Minimum Number of network interfaces (ENIs) that can be attached to the instance If --network-interfaces-max is not specified, the upper bound will be infinity
```
Comment (brycahta) [inline README.md:161]: Only having context from `ByteQuantity` code: 

> If no unit is appended, GiB is assumed

do you want that to be the common theme throughout? If so, then `vcpus-to-memory-ratio` description + logic may need to be updated

File: README.md (hunk 1 above)
Comment (bwagner5) [reply README.md:148]: yep

File: README.md (hunk 2 above)
Comment (bwagner5) [reply README.md:161]: great catch!! This was wrong before too, it was always using GiB for ratio

**Pull request merged**